### ISSUE TITLE
Unable to deploy dash applications in RStudio connect

### ISSUE THREAD

**Issue by edgBR** [rstudio/rsconnect-python#173]
Dear colleagues,

I am using RStudio Connect:
RStudio Connect v1.8.6.1
Build v1.8.6.1-0-ga70e45c

I have created a virtual environment that matches the python version of my RStudio connect server (3.7.6). I have installed the rsconnect-python (1.5.2) and I have successfully added my server an API key:

```
(bmt-dash) edgar@xxxxxx:~/bmt-mockup$ rsconnect add \>     --api-key TJFYUTnCTe6477JZprDgsnNFjj2foabM \
>     --server https://xxxx.xxx/connect/ \
>     --name myserver
Checking https://xxxx.xxx/connect/... [OK]
Verifying API key...                             [OK]
```

However I can not publish my dash application as I am getting this error:

```
(bmt-dash) edgar@xxxxxxx:~/bmt-mockup$ rsconnect deploy dash 
Usage: rsconnect deploy [OPTIONS] COMMAND [ARGS]...
Try "rsconnect deploy --help" for help.

Error: No such command "dash".
(bmt-dash) edgar@xxxxxx:~/bmt-mockup$ rsconnect deploy dash -n myserver
Usage: rsconnect deploy [OPTIONS] COMMAND [ARGS]...
Try "rsconnect deploy --help" for help.
```

I am following this documentation: https://docs.rstudio.com/connect/user/dash/

Does someone knows what is going on?

BR
/Edgar

**Comment by mmarchetti** [comment]
Hi Edgar, sorry for not seeing this sooner. I recommend you open a ticket with RStudio Support to work through this.

**Comment by edgBR** [comment]
Hi @mmarchetti,

We fix it by forcing 1.5.2. We have RSconnect running behind an nginx proxy.

BR
/Edgar
